src/string.rs

```diff
@@ -88,6 +94,34 @@ extern "C" {
   #[allow(dead_code)]
   fn v8__String__IsOneByte(this: *const String) -> bool;
   fn v8__String__ContainsOnlyOneByte(this: *const String) -> bool;
+
+  #[link_name = "_ZTV34ExternalConstOneByteStringResource"]
```

Comment:
Since we can't make this `const`-compatible, I wonder if we should return the vtable pointer itself from C++ code which should be compatible with pretty much any gcc/clang platform

Reply:
That's definitely possible. The only downside is that we'll need to bind a static (maybe AtomicPtr again) that gets initialized by calling into C++, and that call needs to allocate an instance to get the vtable pointer out of it, then delete and return.

One alternative would be to have the `init` call actually call into C++ for in-place building if vtable pointer is not in place yet.

Reply:
Change to calling into C++ for in-place construction.